webapp/pages/admin/users.vue

```diff
@@ -33,6 +33,11 @@
             <b>{{ scope.row.name | truncate(20) }}</b>
           </nuxt-link>
         </template>
+        <template slot="email" slot-scope="scope">
+          <a :href="`mailto:${scope.row.email}`">
+            <b>{{ scope.row.email }}</b>
```

Comment:
Some of those emails are quite long. Maybe we should think about how to fit the data into the layout? E.g. at some places in the app, we use `| truncate(20)` filter.

Although I'm OK to merge this and wait for our admins to complain about it :wink:

Reply:
That's what I thought as well, @datenbrei mentioned that they might be copying and pasting the links, so maybe it's more difficult if it's truncated??

Reply:
Thanks @mattwr18 @roschaefer 
Let me know if I need to update this :D